lib/node_modules/@stdlib/stats/base/ndarray/dmidrange/benchmark/benchmark.js

```diff
@@ -0,0 +1,103 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var bench = require( '@stdlib/bench' );
+var uniform = require( '@stdlib/random/base/uniform' );
+var filledarrayBy = require( '@stdlib/array/filled-by' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var pow = require( '@stdlib/math/base/special/pow' );
+var ndarray = require( '@stdlib/ndarray/base/ctor' );
+var pkg = require( './../package.json' ).name;
+var dmidrange = require( './../lib' );
+
+
+// FUNCTIONS //
+
+/**
+* Returns a random number (80% chance of returning a `NaN`).
+*
+* @private
+* @returns {number} random number or `NaN`
+*/
+function rand() {
+	return uniform( -10.0, 10.0 );
+}
+
+/**
+* Creates a benchmark function.
+*
+* @private
+* @param {PositiveInteger} len - array length
+* @returns {Function} benchmark function
+*/
+function createBenchmark( len ) {
+	var xbuf;
+	var x;
+
+	xbuf = filledarrayBy( len, 'float64', rand );
```

Comment:
You don't need to use `filledarrayBy` and `new ndarray` here. You can use `random/uniform` directly.